config/application.yml.default

```diff
@@ -139,7 +139,6 @@ in_person_email_reminder_early_benchmark_in_days: 11
 in_person_email_reminder_final_benchmark_in_days: 1
 in_person_email_reminder_late_benchmark_in_days: 4
 in_person_proofing_enabled: false
-in_person_send_proofing_notifications_enabled: false
```

Comment:
IIRC the application is likely to encounter an error if it attempts to access this configuration and it's not defined the the configuration file. I would suggest keeping this b/c it is currently the default configuration for deployed environments.

Reply:
agree.

Reply:
it was defined twice in the default file, so I removed one invocation

Reply:
Hmm, someone else added a copy 👍